servers/src/main/java/tachyon/master/file/meta/Inode.java

```diff
@@ -197,12 +207,58 @@ public synchronized void setPinned(boolean pinned) {
     mPinned = pinned;
   }
 
+  /**
+   * Get the username of the inode
+   */
+  public synchronized String getUsername() {
+    return mUsername;
+  }
+
+  /**
+   * Set the username of the inode
```

Comment:
NIT: `Sets the username of the inode.` to be consistent with other parts of this file. Also for the following javadocs